mongodb-advanced/index.js

```diff
@@ -27,8 +28,12 @@ bedrock.events.on('bedrock-express.configure.routes', addRoutes);
 
 function addRoutes(app) {
   app.get('/people', (req, res) => {
-    database.collections.mongodb_advanced.find({}).toArray(function(err, docs) {
+    database.collections.mongodb_advanced.find({}).toArray((err, docs) => {
       res.send(docs);
+    }, err => {
```

Comment:
you have incorrectly created a second arrow function as a second parameter to `toArray`, the conditional here nees to move to L32.